### DISCUSSION TITLE
Enable SSO button is not showing up in the Personal Access Token.

### DISCUSSION THREAD

**Discussion by SAEb-ai** [razorpay/blade#397] [Q&A] [answered]
Hi everyone, I am excited to contribute to the blade repository of Razorpay. But I am stuck at one of the [Pre-requisite points](https://github.com/razorpay/blade#-pre-requisite) (Generate a Personal Access Token on GitHub and enable SSO).

I have created the Personal Access Token but the enable SSO button is not coming up in the token.
![razor](https://user-images.githubusercontent.com/69013647/147214736-ea0707f2-b87b-47b2-9785-e6f330333103.JPG)

Found a [topic](https://github.community/t/trouble-authorizing-single-sign-on-authorization/1698/4) in the Github Support related to this problem where the discussion was mostly related to organization. So, I just wanted enquire from you all how have you been able to tackle this!!

Hoping to hear from you soon,

Edit: Friendly ping @kamleshchandnani.

Thanks and Regards,
Md Shahbaz Alam

**Comment by kamleshchandnani** [comment, marked as answer]
@SAEb-ai So sorry that's a miss on our part. That enable SSO is only available for users that are part of `razorpay` org. I've raised a [PR](https://github.com/razorpay/blade/pull/396/files#diff-85b67067082bd8b5f355f28c1ca89771de3972098700bc800e6f7d2b7febf24cR40) to accommodate this change. Meanwhile, can you try if you can install packages without the SSO thing?
